demisto_sdk/commands/common/tests/integration_test.py

```diff
@@ -190,6 +190,35 @@ def test_is_changed_command_name_or_arg(self, current, old, answer):
         structure.quiet_bc = True
         assert validator.is_changed_command_name_or_arg() is False  # if quiet_bc is true should always succeed
 
+    CHANGED_COMMAND_OR_ARG_MST_TEST_INPUTS = [
+        ([{"name": "test", "arguments": [{"name": "test1"}]}, {"name": "test2", "arguments": [{"name": "test3"}]}],
+         [{"name": "test", "arguments": [{"name": "test"}, {"name": "test1", "required": True}]},
+         {"name": "test2", "arguments": [{"name": "test2"}, {"name": "test3", "required": True}]}],
+         "[ERROR]: : [BC104] - Possible backwards compatibility break, You've changed the name of some of the command(s) or its arg in the file,"
+         " please undo, the following commands:\ntest\ntest2")
+    ]
+
+    @pytest.mark.parametrize("current, old, expected_error_msg", CHANGED_COMMAND_OR_ARG_MST_TEST_INPUTS)
+    def test_is_changed_command_name_or_arg_msg(self, capsys, current, old, expected_error_msg):
+        """
+        Given
+        - Case 1: previous yml content with 2 commands and current yml content with the 2 commands with name
```

Comment:
The 'given' information is missing some of the data and it is not clear. When you write a 'given' section in a test try to be clear and concise, for example:
```python
"""
Given:
    Case 1: An integration with BC break in the following way:
        Old copy: Has two commands - command_test_name_1 and command_test_name_2 where each command has 2 arguments - argument_test_name_1, argument_test_name_2, argument_test_name_3, argument_test_name_4 respectively.
        New copy: Has two commands - command_test_name_1 and command_test_name_2 where each command has 1 arguments - argument_test_name_1 and argument_test_name_3 respectively.
"""
```